packages/fluentui/docs/package.json

```diff
@@ -44,7 +44,9 @@
     "react-source-render": "4.0.0-1",
     "react-textarea-autosize": "7.0.0",
     "react-virtualized": "^9.21.1",
-    "react-vis": "^1.11.6"
+    "react-vis": "^1.11.6",
+    "react-window": "^1.8.5",
+    "what-input": "^5.2.10"
```

Comment:
if I remember correctly, we ship our own implementation of what-input, can you use that instead?

Reply:
What-input is removed based on the discussion